### PR TITLE
Order by oldest first [WIP]

### PR DESCRIPTION
**Pull request by m0g**

### CODE REVIEW

File: js/theme/handlers.js [hunk 1]
```diff
@@ -19,6 +19,20 @@ var buttons = {
...
+    },
+    "[data-js-orderby-oldest-first]": function() {
+      viewmodel.loadPosts({sort: 'oldest_first', returnPromise: true})
+        .then(view.renderTimeline)
+        .then(function() {
+          view.toggleSortBtn('oldest_first')
+        })
+    },
+    "[data-js-orderby-newest-first]": function() {
+      viewmodel.loadPosts({sort: 'newest_first', returnPromise: true})
```
Comment (psolbach) [inline js/theme/handlers.js]: As we discussed, the returnPromise conditional is a workaround. I will see what's going on and propose we not introduce that logic here.

File: templates/content.html [hunk 2]
```diff
@@ -5,8 +5,16 @@
         <div class="header-bar">
             <div class="sorting-bar">
                 <div class="sorting-bar__orders">
-                    <div class="sorting-bar__order sorting-bar__order--active" onClick="timeline.orderBy('newest_first')">{{ theme_options.l10n.newestFirst }}</div>
-                    <div class="sorting-bar__order" onClick="timeline.orderBy('oldest_first')">{{ theme_options.l10n.oldestFirst }}</div>
+                    <div
+                      id="newest_first"
```
Comment (psolbach) [inline templates/content.html]: Why use id's here?

File: js/theme/viewmodel.js [hunk 3]
```diff
@@ -114,7 +122,10 @@ function renderPosts(api_response, opts) {
  * @param {object} api_response - liveblog API response JSON.
  */
 function updateViewModel(api_response, opts) {
-  vm._items.push.apply(vm._items, api_response._items);
+  if (opts.sort === 'oldest_first')
```
Comment (psolbach) [inline js/theme/viewmodel.js]: Sticking with the Crockford way of doing things, we should use curly braces here (like you did elsewhere: https://github.com/liveblog/liveblog-default-theme/blob/67e6dedcee27b66db0743d9991af397080ffac6f/js/theme/helpers.js#L60).

File: js/theme/view.js [hunk 4]
```diff
@@ -4,10 +4,41 @@
...
+  renderedPosts.forEach(function(renderedPost) {
+    timelineElem[0].innerHTML += renderedPost;
+  });
+}
+
+function toggleSortBtn(name) {
+  document
+    .querySelectorAll('.sorting-bar__order')
+    .forEach(function(btn) {
+      btn.classList.remove('sorting-bar__order--active');
```
Comment (psolbach) [inline js/theme/view.js]: I suggest `classList.toggle`

File: js/theme/viewmodel.js (hunk 3 above)
Comment (m0g) [inline js/theme/viewmodel.js]: Sure, maybe we could set up some linting. The only problem being that the current style adopted is quite different from the sourcefabric one

File: js/theme/handlers.js (hunk 1 above)
Comment (mscam) [inline js/theme/handlers.js]: Let's create an issue for that, we can get this PR approved in the meanwhile.

**Pull request merged**